charts/stable/emby/values.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: ghcr.io/k8s-at-home/emby
+  pullPolicy: IfNotPresent
+  tag: v4.5.4.0
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the emby documentation
+# https://emby.org/docs
+env: {}
+  # TZ:
+
+service:
+  port:
+    port: 8096
+  additionalPorts:
+    - port: 8920
+      name: https
+      protocol: TCP
+
+ingress:
+  enabled: false
```

Comment:
You'll probably want to include some persistence below here for config and media.

Reply:
I'm going to have to look into this a bit more. [Truecharts'](https://github.com/truecharts/apps/blob/56211222bff4333fdf827d34a88c62965777277c/.containers/apps/emby/Dockerfile) didn't have any `VOLUMES` , [LSIO](https://github.com/linuxserver/docker-emby/blob/master/README.md#usage) has `/data` but I don't see how Emby is mapped to it, and the official Emby Docker file is only calling out `/mnt/` but are [not referencing](https://github.com/MediaBrowser/Emby.Build/tree/master/docker-containers) them.

```
 volumes:
      - /path/to/programdata:/config # Configuration directory
      - /path/to/tvshows:/mnt/share1 # Media directory
      - /path/to/movies:/mnt/share2 # Media directory
```

Reply:
It looks like /media already exists in the image and the storage locations are set using the emby GUI. I've also added the `/config` volume. Volume mounts seem to work without declaring `VOLUME` in the Dockerfile and so I don't think there's a need to update the image.

Reply:
Looks good man, thanks!